src/Tools/dotnet-user-jwts/src/Commands/CreateCommand.cs

```diff
@@ -123,7 +123,7 @@ public static void Register(ProjectCommandLineApplication app)
         var name = nameOption.HasValue() ? nameOption.Value() : Environment.UserName;
         optionsString += $"{Resources.JwtPrint_Name}: {name}{Environment.NewLine}";
 
-        var audience = audienceOption.HasValue() ? audienceOption.Values : DevJwtCliHelpers.GetAudienceCandidatesFromLaunchSettings(project).ToList();
+        var audience = audienceOption.HasValue() ? audienceOption.Values : DevJwtCliHelpers.GetAudienceCandidatesFromLaunchSettings(project)?.ToList();
         optionsString += audienceOption.HasValue() ? $"{Resources.JwtPrint_Audiences}: {audience}{Environment.NewLine}" : string.Empty;
```

Comment:
```suggestion
        optionsString += audience is not null ? $"{Resources.JwtPrint_Audiences}: {audience}{Environment.NewLine}" : string.Empty;
```
This would then print of the URL gotten from launchSettings which we weren't including before (bug creep 😆)

Reply:
That was actually intentional. This was meant to only display options that were explicitly passed by the user, not ones we implicitly derived (e.g. username from `Environment.UserName` or urls from launchSettings).